### PR TITLE
fix(settings): Update Europe/Kiev to correct timezone

### PR DESCRIPTION
**Pull request by silent1mezzo**
Kiev's correct timezone is UTC+0300 (https://www.timeanddate.com/worldclock/ukraine/kyiv)

Moment.js already uses UTC+0300
Screen Shot 2020-06-15 at 10 57 16 PM

### CODE REVIEW

**Comment by silent1mezzo** [comment]
@billyvg Kiev does have daylight savings where it's UTC+0200 for 5 months of the year so this change makes it more consistent for the majority of the year it doesn't actually solve the problem during the winter months (when UTC+0200 is correct)

**Comment by silent1mezzo** [comment]
Our version of pytz (2018.9) also shows Kiev as UTC+0300
Screen Shot 2020-06-16 at 9 15 29 PM

File: src/sentry/static/sentry/app/data/timezones.tsx [hunk 1]
```diff
@@ -293,6 +292,7 @@ export default [
   ['Africa/Djibouti', '(UTC+0300) Africa/Djibouti'],
   ['Africa/Juba', '(UTC+0300) Africa/Juba'],
   ['Africa/Kampala', '(UTC+0300) Africa/Kampala'],
+  ['Europe/Kiev', '(UTC+0300) Europe/Kiev'],
```
Comment (billyvg) [inline src/sentry/static/sentry/app/data/timezones.tsx:295]: nit: Can we group this will the other Europe/UTC+0300 entries?

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/data/timezones.tsx (hunk 1 above)
Comment (silent1mezzo) [reply src/sentry/static/sentry/app/data/timezones.tsx:295]: @billyvg totally agree with that. I grouped it using K in Kiev ignoring the region for some reason. Updated.

**Review by priscilawebdev — APPROVED** [review]

**Pull request merged**